### PR TITLE
Adding ActiveAdmin, Devise and Kaminari gems

### PR DESCRIPTION
**Pull request by NikolayGGorbunov**
Also add sorting by columns and update index via Devise

### CODE REVIEW

File: spec/controllers/packages_controller_spec.rb [hunk 1]
```diff
@@ -3,40 +3,127 @@
...
+      end
+    end
+
+    context 'when packages first pack > second by price (reverse)' do
+      let(:sort_params) { { direction: 'desc', sort: 'updated_at' } }
+
+      it "sort correctly" do
+        sort
+        expect(assigns(:packages)).to eq([user0.packages.last, user0.packages.first])
+      end
```
Comment (Sh1mensoka) [inline spec/controllers/packages_controller_spec.rb:105]: Do we really need these tests if we use same methods with different params?

File: spec/controllers/packages_controller_spec.rb [hunk 2]
```diff
@@ -51,9 +138,21 @@
...
+      sign_in user0
+    end
+
+    context 'when user sign out' do
+      before do
+        sign_out user0
+      end
+      subject(:pack_create) { process :create, method: :post, params: { package: params } }
+      include_examples 'when log out'
+    end
```
Comment (Sh1mensoka) [inline spec/controllers/packages_controller_spec.rb:154]: Are you sure that package won't be created if user signed out? Why don't we check it?

File: spec/support/shared_examples.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+
+#RSpec.shared_examples 'when log out' do |subj, usr|
+#  it 'redirect to sign in' do
+#    binding.pry
+#    sign_out usr
+#    expect(subject).to redirect_to(new_user_session_path)
+#  end
+#end
```
Comment (Sh1mensoka) [inline spec/support/shared_examples.rb:8]: If we don't need this file we should remove it

Comment (NikolayGGorbunov) [reply spec/support/shared_examples.rb:8]: Fixed

File: spec/controllers/packages_controller_spec.rb (hunk 1 above)
Comment (NikolayGGorbunov) [reply spec/controllers/packages_controller_spec.rb:105]: Not too much

File: spec/controllers/packages_controller_spec.rb (hunk 2 above)
Comment (NikolayGGorbunov) [reply spec/controllers/packages_controller_spec.rb:154]: Didn't think about it. Added

**Review by NadezhdaGlin — APPROVED** [review]

**Review by Sh1mensoka — APPROVED** [review]

**Pull request merged**